Modules/Office365DSC/DSCResources/MSFT_SPOAccessControlSettings/MSFT_SPOAccessControlSettings.psm1

```diff
@@ -0,0 +1,349 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param (
+
+        [Parameter(Mandatory = $true)]
+        [ValidateSet('Yes')]
+        [String]
+        $IsSingleInstance,
+
+        [Parameter()]
+        [System.Boolean]
+        $DisplayStartASiteOption,
+
+        [Parameter()]
+        [System.String]
+        $StartASiteFormUrl,
+
+        [Parameter()]
+        [System.Boolean]
+        $IPAddressEnforcement,
+
+        [Parameter()]
+        [System.String]
+        $IPAddressAllowList,
+
+        [Parameter()]
+        [System.UInt32]
+        $IPAddressWACTokenLifetime,
+
+        [Parameter()]
+        [System.Boolean]
+        $CommentsOnSitePagesDisabled,
+
+        [Parameter()]
+        [System.Boolean]
+        $SocialBarOnSitePagesDisabled,
+
+        [Parameter()]
+        [System.Boolean]
+        $DisallowInfectedFileDownload,
+
+        [Parameter()]
+        [System.boolean]
+        $ExternalServicesEnabled,
+
+        [Parameter()]
+        [System.Boolean]
+        $EmailAttestationRequired,
+
+        [Parameter()]
+        [System.UInt32]
+        $EmailAttestationReAuthDays,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $CentralAdminUrl,
+
+        [Parameter(Mandatory = $true)]
+        [System.Management.Automation.PSCredential]
+        $GlobalAdminAccount
+    )
+    
+    Test-PnPOnlineConnection -SPOCentralAdminUrl $CentralAdminUrl -GlobalAdminAccount $GlobalAdminAccount
+
+    $nullReturn = @{
+        IsSingleInstance             = 'Yes'
+        DisplayStartASiteOption      = $null
+        StartASiteFormUrl            = $null
+        IPAddressEnforcement         = $null
+        IPAddressAllowList           = $null
+        IPAddressWACTokenLifetime    = $null
+        CommentsOnSitePagesDisabled  = $null
+        SocialBarOnSitePagesDisabled = $null
+        DisallowInfectedFileDownload = $null
+        ExternalServicesEnabled      = $null
+        EmailAttestationRequired     = $null
+        EmailAttestationReAuthDays   = $null
```

Comment:
Need to also return GlobalAdminAccount and CentralAdminUrl from the PSBoundParameters directly